app/assets/stylesheets/administrate/library/_data_label.scss

```diff
@@ -1,6 +1,6 @@
 @mixin data-label {
   color: $hint-grey;
-  font-size: modular-scale(-1);
+  font-size: 1em / 1.2em;
```

Comment:
Can we put a raw value here that closely matches what the output of `modular-scale` was, instead of math?